caddy/restartinproc.go

```diff
@@ -23,6 +26,7 @@ func restartInProc(newCaddyfile Input) error {
 		} else {
 			wg.Done() // take down our barrier
 		}
+		go notify.IsReloading(false).Tell()
```

Comment:
Good one. I thought that at first, too. But it's already covered by the ```err = Start()``` in new line 21, which already results in a overriding notification.